modules/end-user-guide/partials/assembly_creating-a-workspace-from-code-sample.adoc

```diff
@@ -19,10 +19,6 @@ For more information about devfiles, see xref:configuring-a-workspace-using-a-de
 
 include::partial$proc_creating-a-workspace-from-user-dashboard-get-started.adoc[leveloffset=+1]
 
-include::partial$proc_creating-a-workspace-from-user-dashboard.adoc[leveloffset=+1]
-
-include::partial$proc_changing-the-configuration-of-an-existing-workspace.adoc[leveloffset=+1]
-
-include::partial$assembly_running-an-existing-workspace-from-the-user-dashboard.adoc[leveloffset=+1]
+include::partial$proc_creating-a-custom-workspace-from-a-template.adoc[leveloffset=+1]
```

Comment:
The content above should be updated accordingly 
```
Every stack includes a sample codebase, which is defined by the devfile of the stack. This section explains how to create a workspace from this code sample in a sequence of three procedures.
. Creating a workspace from the user dashboard:
.. Using the xref:creating-a-workspace-from-get-started-view-of-user-dashboard_{context}[Get Started view].
.. Using the xref:creating-a-workspace-from-custom-workspace-view-of-user-dashboard_{context}[Custom Workspace view].
. xref:changing-the-configuration-of-an-existing-workspace_{context}[Changing the configuration of the workspace] to add code sample.
. xref:running-an-existing-workspace-from-the-user-dashboard_{context}[Running an existing workspace from the user dashboard].
```

Reply:
Is fixed in https://github.com/eclipse/che-docs/pull/1737/commits/a05793bc6cea3445add584ce791ca2bb641fc802
@MichalMaler please verify

Reply:
Done